tests/test_admin.py

```diff
@@ -9,11 +9,35 @@
 from .models import Author, Publisher, Book, Library
 
 
+def load_fixtures():
```

Comment:
Jut curious, would we ever load fixtures from xml instead of doing it inline like this?